src/client/Microsoft.Identity.Client/TelemetryCore/TelemetryClient/TelemetryDatapoints.cs

```diff
@@ -0,0 +1,23 @@
+﻿// Copyright (c) Microsoft Corporation. All rights reserved.
+// Licensed under the MIT License.
+
+using Microsoft.IdentityModel.Abstractions;
+
+namespace Microsoft.Identity.Client.TelemetryCore.TelemetryClient
+{
+    /// <summary>
+    /// Stores the cache details to log to <see cref="ITelemetryClient"/>.
+    /// </summary>
+    public class TelemetryDatapoints
+    {
+        /// <summary>
+        /// Total latency of L1 cache access. This data is captured in MSAL when accessing the internal cache or Microsoft.Identity.Web when accessing the memory cache.
+        /// </summary>
+        public long L1Latency { get; set; }
+
+        /// <summary>
+        /// Total latency of L2 cache access. This data is captured in Microsoft.Identity.Web when accessing the distributed cache.
+        /// </summary>
+        public long L2Latency { get; set; }
```

Comment:
I think we've talked before about whether L1 and L2 latency needs to be stored separately. In my conception it's just latency and then we'd log what the token source was to inform what kind of latency. I don't think we'd ever have the case where we found a token in L1 and still looked in L2 and if we have to look in L2 it should include the L1 look up since that's part of the latency e2e (imo).

SEAL folks seemed to be of the same opinion in "MSAL's schema logged to telemetry pipeline" thread.

Reply:
Ack, @neha-bhargava  is pausing this until we have a clearer picture of the schema. Will discuss with E&D team on Tuesday. Transforming this into a draft PR.